extensions/interactions/base.py

```diff
@@ -263,3 +263,18 @@ def get_rule_param_type(self, rule_name, rule_param_name):
                 return param_type
         raise Exception(
             'Rule %s has no param called %s' % (rule_name, rule_param_name))
+
+    def get_hint_proto_list(self, hints):
+        """Gets the list of HintDto proto for given hints domain object list.
+
+        Args:
+            hints: list(Hint). List of hints.
+
+        Returns:
+            list. The list of HintDto proto object.
```

Comment:
Usually our typeinfo for lists is in the form `list(...)` (where you'll need to fill in the type for the `...`)

Reply:
Fixed